src/utils/verification.js

```diff
@@ -0,0 +1,53 @@
+const ver1 = require("../constants/verification");
```

Comment:
```suggestion
const {verificationStatus} = require("../constants/verification");
```

이렇게 하면 임시 변수로 할당하지 않아도 됨. 그리고 `ver1`같이 의미가 모호한 변수는 절대 사용하지 마시고.. 차라리 불러온다고하면 이렇게 하던지
```ts
const verification = require("../constants/verification");
```

Reply:
수정하겠음!